### DISCUSSION TITLE
Text Generation w/o repeating the prompt

### DISCUSSION THREAD

**Discussion by wildmanwong** [vllm-project/vllm#1322] [Q&A]
Is there a parameter to specify if we do NOT want the prompt repeated when generating text?

**Comment by AguirreNicolas** [comment]
You should modify this line probably.
https://github.com/vllm-project/vllm/blob/7b895c5976a81cb1428b24b56fcc4d3716b53710/vllm/entrypoints/api_server.py#L61

Or on the client side take len(prompt) and remove then from the response.
